jpo-ode-common/src/main/java/us/dot/its/jpo/ode/util/JsonUtils.java

```diff
@@ -36,172 +34,300 @@
 import lombok.extern.slf4j.Slf4j;
 import org.json.JSONObject;
 
+/**
+ * Utility class for JSON operations using Jackson ObjectMapper.
+ */
 @Slf4j
 public class JsonUtils {
 
```

Comment:
Primarily formatting changes to address checkstyle violations. A plain object mapper `mapper_plain` was added along with a getter helper method. This is used in the `OdeMessageFrameDataCreatorHelper` class to serialize message frames.